python/py-jsonschema/Portfile

```diff
@@ -22,47 +22,42 @@ long_description    {*}${description}
 
 homepage            https://github.com/python-jsonschema/jsonschema
 
-checksums           rmd160  86637e335ef60176700311eb93e0d5dbbcb8d457 \
-                    sha256  0f864437ab8b6076ba6707453ef8f98a6a0d512a80e93f8abdb676f737ecb60d \
-                    size    297785
+checksums           rmd160  9344c735429814aaf4d9a3fa036605053ccfbd2a \
+                    sha256  ec84cc37cfa703ef7cd4928db24f9cb31428a5d0fa77747b8b51a847458e0bbf \
+                    size    318089
 
 if {${name} ne ${subport}} {
-    depends_build-append \
-                        port:py${python.version}-setuptools_scm \
-                        port:py${python.version}-hatch-vcs \
-                        port:py${python.version}-hatch-fancy-pypi-readme
-
-    # CLI depends on pkg_resources.py from setuptools
-    depends_lib-append  port:py${python.version}-attrs \
-                        port:py${python.version}-pyrsistent \
-                        port:py${python.version}-setuptools
-
-    if {${python.version} < 37} {
+    if {${python.version} < 38} {
         version             3.2.0
         revision            0
         distname            ${python.rootname}-${version}
         checksums           rmd160  fb79174fba8955cf5433fa0e79d3ad3a48c590fb \
                             sha256  c8a85b28d377cc7737e46e2d9f2b4f44ee3c0e1deac6bf46ddefc7187d30797a \
                             size    167226
         python.pep517       no
-        depends_build-delete \
-                        port:py${python.version}-hatch-vcs \
-                        port:py${python.version}-hatch-fancy-pypi-readme
-        depends_lib-append  port:py${python.version}-six
+
+        depends_build-append  port:py${python.version}-setuptools_scm
+        depends_lib-append    port:py${python.version}-attrs \
+                              port:py${python.version}-importlib-metadata \
+                              port:py${python.version}-pyrsistent \
+                              port:py${python.version}-setuptools \
+                              port:py${python.version}-six
     } else {
-        patchfiles-append   patch-setup.py.diff
-        if {${python.version} < 38} {
-            depends_lib-append  port:py${python.version}-typing_extensions
-        }
+        # CLI depends on pkg_resources.py from setuptools
+
+        depends_build-append  port:py${python.version}-hatch-vcs \
+                              port:py${python.version}-hatch-fancy-pypi-readme
+
+        depends_lib-append    port:py${python.version}-attrs \
+                              port:py${python.version}-jsonschema_specifications \
+                              port:py${python.version}-referencing \
+                              port:py${python.version}-rpds_py
+
```

Comment:
please verify that everything is aligned with "four spaces"; it doesn't look like the dependencies here are